src/Deposit.js

```diff
@@ -687,7 +733,12 @@ export default class Deposit {
 
     ///------------------------------- Helpers ---------------------------------
 
-    // autoSubmitting/*: boolean*/
+    /**
+     * @typedef {Object} AutoSubmitState
+     * @prop {Promise<BitcoinTransaction>} fundingTransaction
+     * @prop {Promise<{ transaction: FoundTransaction, requiredConfirmations: Number }>} fundingConfirmations
```

Comment:
Tiny observation - exposing `requiredConfirmations` *after* we've waited those confirmations feels a bit confusing at first.

Reply:
Yeah… I file this under “necessary but annoying”, because the next step (proof) uses the required confirmations to limit the SPV proof to the required amount of work, rather than submitting all work on the transaction. That lets us avoid issues with submitting older transactions whose proof data ends up being too large.